tests/test_Storable.py

```diff
@@ -148,6 +153,69 @@ def testNewValue(self):
         cppLib.assertCppValue(self.testbed, newstr)
 
 
+@dataclass
+class Blob(Storable):
```

Comment:
What does this class add that `DemoStorable`, above, does not?

Reply:
That isn't persistable, afaict.

Reply:
Could it be made persistable? Or does Python-side support for `Persistable` require something (e.g., `@dataclass`) that's incompatible with the existing class?

Reply:
I suppose it can, but I also thought it was useful to have an example here that was a straight dataclass with yaml persistence...it's a bit redundant, is that a problem?